azure/components/apigateway/id.ftl

```diff
@@ -1,13 +1,46 @@
 [#ftl]
+[#include "openapi.ftl"]
 
 [@addResourceGroupInformation
     type=APIGATEWAY_COMPONENT_TYPE
-    attributes=[]
+    attributes=[
+        {
+            "Names" : "ContentType",
+            "Description" : "The content type of the API's schema",
+            "Type" : STRING_TYPE,
+            "Default" : "application/vnd.oai.openapi.components+json"
```

Comment:
this only has two potential values right? json and yaml?

Reply:
Removed as no longer required